PendingReleaseNotes.md

```diff
@@ -6,6 +6,7 @@
 - Creation of storage pools through the custom resource definitions (CRDs) now allows users to optionally specify `deviceClass` property to enable
 distribution of the data only across the specified device class. See [Ceph Block Pool CRD](Documentation/ceph-pool-crd.md#ceph-block-pool-crd) for
 an example usage
+- Added support for K8s 1.15 and removed support for K8s 1.10.
```

Comment:
I think it's a bit misleading, the only thing this PR does is to remove CI coverage on 1.10 and add a CI test on 1.15 with no guarantee that 1.15 will actually work.

Reply:
Description is updated. Still, I would assert that whatever is in the CI test matrix is what is officially supported.